po/de.po

```diff
@@ -4280,14 +4278,14 @@ msgid "later"
 msgstr "später"
 
 #: ../src/common/exif.cc:3980
-#, fuzzy, c-format
+#, c-format
 msgid "cannot read xmp file '%s': '%s'"
-msgstr "konnte xmp Datei „%s” nicht lesen: „%s\""
+msgstr "konnte xmp Datei '%s’ nicht lesen: ‘%s’"
```

Comment:
Why this change? The typographically correct quotaion marks should be used here.
"konnte xmp Datei „%s” nicht lesen: „%s”"